raiden/blockchain/events.py

```diff
@@ -144,7 +147,7 @@ def get_all_netting_channel_events(
         pyethapp_chain,
         netting_channel_address,
         events=ALL_EVENTS,
-        from_block=0,
+        from_block='earliest',
```

Comment:
Right, going back to `from_block=0`, although this default value isn't used a lot: calls coming from API endpoints always put `None` here, so a truth check down the line (as I did above) will be the actual codepath. Thanks.